src/components/mediaManager/AposMediaManagerDisplay.vue

```diff
@@ -19,16 +19,21 @@
       <div
         class="apos-media-manager-display__cell" v-for="item in media"
         :key="generateId(item.id)"
-        :class="{'is-selected': !!item.checkbox.value.data.length}"
+        :class="{'is-selected': checked.includes(item.id)}"
       >
         <div class="apos-media-manager-display__checkbox">
           <AposCheckbox
             tabindex="-1"
-            :field="item.checkbox.field"
-            :value="item.checkbox.value.data"
-            :status="item.checkbox.status"
-            :choice="item.checkbox.choice"
-            @toggle="$emit('select-another', item.id)"
+            :field="{
```

Comment:
Since I got rid of `myMedia`, I moved the fairly simple checkbox construction here.